docs/cloud/rahti/rahti-migration.md

```diff
@@ -211,3 +211,7 @@ In the Project details page (`Developer` > `Project`), click `PersistentVolumeCl
 
 !!! warning "Lazy volume creation"
     The volume will only be created when it is mounted for the first time, this is a change in behavior in `Rahti 2`.
+
+### How to use integrated registry
```

Comment:
Should Integrated Registry be with capital letters? At least later on you have used capital letters. Or then stick with lower case letters thorough the document.

Reply:
Fixed